src/gfx/render.js

```diff
@@ -218,6 +231,7 @@ var ballsAndSticksForChain = (function() {
     meshGeom.addIdRange(idRange);
     // generate geometry for each atom
     chain.eachAtom(function(atom) {
+      var atomRadius = opts.sphereRadius * VDW_RADIUS[atom._element];
```

Comment:
You should use atom.element() instead of using the underscored atom._element here. I believe this will fail when calling ballsAndSticks on a view, e.g. ``viewer.ballsAndSticks('v', structure.select({chain: 'A'}));``

Also, atomRadius will be NaN if the atom either doesn't have a proper element or the element is not contained in the VDW_RADIUS map. There should be a proper fallback for this case as otherwise the sphere won't  be drawn properly.